src/utils/dates.ts

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright 2023 Cortex Applications, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { isNil } from 'lodash';
+import moment from 'moment/moment';
+
+export const daysUntil = (expirationDate?: string) => {
+  return !isNil(expirationDate)
+    ? moment(expirationDate).diff(moment(), 'days')
```

Comment:
🤔 Do we need to worry about TZ for these? I would assume our backend returns UTC dates; if so shouldn't this be
```suggestion
    ? moment(expirationDate).diff(moment.utc(), 'days')
```
?

Reply:
TZ is like "different presentation mode". If in 7 hours earth will explode, it will happen in ~7 hours both for me, you and some guy in Greenwich, no matter what hour will our clock show.

Reply:
OK but if you are deriving "7 hours" from comparing 700 EST vs 1400 CET then it's not actually 7 hours...